flink-runtime/src/test/java/org/apache/flink/runtime/source/coordinator/SourceCoordinatorContextTest.java

```diff
@@ -71,16 +72,16 @@ public void testUnregisterReader() {
 	}
 
 	@Test
-	public void testAssignSplitsFromCoordinatorExecutor() throws ExecutionException, InterruptedException {
+	public void testAssignSplitsFromCoordinatorExecutor() throws ExecutionException, InterruptedException, IOException {
```

Comment:
Nit: For tests it is often easier to just declare `throws Exception`, rather than have a long list of individual exceptions.